bin/clone

```diff
@@ -45,15 +44,24 @@ def main():
     else:
         log_file = "/opt/snare/clone.log"
     logger.Logger.create_clone_logger(log_file, __package__)
-    print_color("Logs will be stored in {}\n".format(log_file), 'INFO')
+    print("  Logs will be stored in {}".format(log_file), end="")
```

Comment:
why you need empty end here? and space in the beginning?

Reply:
This is for the animation -- 
![image](https://user-images.githubusercontent.com/40897792/76002247-50fa4480-5f2c-11ea-8f18-d6bcf698baf9.png)
Let me know if this looks ok


+ I will also add the print_color functionality to them as well.